src/KafkaFlow.Retry.SqlServer/SqlServerDbSettings.cs

```diff
@@ -1,22 +1,25 @@
 ﻿namespace KafkaFlow.Retry.SqlServer
 {
-    using Dawn;
     using System.Diagnostics.CodeAnalysis;
+    using Dawn;
 
     [ExcludeFromCodeCoverage]
     public class SqlServerDbSettings
     {
-        public SqlServerDbSettings(string connectionString, string databaseName)
+        public SqlServerDbSettings(string connectionString, string databaseName, string schema = "dbo")
```

Comment:
Maybe you should remove this default value and use it in the overload of `WithSqlServerDataProvider(...)` as previously suggested.